snapshots/pom.xml

```diff
@@ -34,6 +34,7 @@
 
     <modules>
         <module>jbatch</module>
+        <module>expressly</module>
```

Comment:
Could the execution at https://github.com/eclipse-ee4j/glassfish/blob/8796647bb57e70610b79bc85f4a0d5f35dbbe186/snapshots/pom.xml#L58 be renamed?

Reply:
Mea culpa :)